src/user/relationship-model.md

```diff
@@ -71,3 +71,15 @@ One important thing to note about bidirectional relationships, are that the
 "from" and "to" sides are still significant in the database, meaning each entity
 must match the constraint for that side. However, from a user perspective, which
 side each entity is stored as is insignificant.
+
+## Example { #relationship_model_example }
+
+<img src="resources/images/relationship_model/relationships.png" width="50%"/>
```

Comment:
Did you choose the `img` tag to specify the `width`? This seems to be the first usage. Other images are added like

https://github.com/dhis2/dhis2-docs/commit/156874b891f2b996061aca7bcf9b9dbd30003a47

https://docs.dhis2.org/en/use/user-guides/dhis-core-version-master/working-with-your-account/messaging.html?h=define+who+you+want+to+receive+the+message+master#create-a-message